feat: display policy's exclude entries

danlucian opened the pull request:
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/CONTRIBUTING.md) rules

#### What does this PR do?

- include the 'exclude' rules on the demunge implementation
- exclude might contain 2 categories/ids: `global` and `code`
- the entries for `exclude.global/code` might be simple paths (strings) or objects (including metadata like reason, creation/expiration dates).

The changes on the policy library were done here: https://github.com/snyk/policy/pull/70

#### Screenshots

<img width="1054" alt="Screenshot 2022-03-24 at 18 21 55" src="https://user-images.githubusercontent.com/6909300/159962982-3347a63f-8576-48e7-84b8-107cbf458ece.png">

dekelund commented on src/lib/display-policy.ts:
```diff
@@ -15,17 +15,32 @@ export async function display(policy) {
+
+    const formattedTitle =
+      title === 'Exclude'
+        ? chalk.bold('\n#' + i + ' ' + title) +
```
would it be possible to make use of backticket as formatting string instead?

danlucian replied:
Sure thing! Changed to template literals.

danlucian commented:
> I can approve the change to the IaC fixture here, but wanted to make sure you're aware of the Snyk IaC functionality under the `exclude` field that was introduced here #3018. It doesn't look from the description like we've tested what the output of this looks like?
> 
> These excluded fields are not file paths but cloud resources, so the text may need to be adapted.

In order to support also the iac-drift entries I updated the message that we use for printing out the items to something like  `Exclude the following <entry_name> items/paths`.

Below is an example:

`.snyk` policy file used as input:

```
# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
# ignores vulnerabilities until expiry date; change duration by modifying expiry date
ignore:
exclude:
  global:
    - some/excluded/file/file-to-exclude.cpp:
        reason: False positive
        expires: 2022-04-17T12:57:47.569Z
        created: 2022-03-18T12:57:47.576Z
    - some/excluded/folder:
        reason: None Given
        expires: 2022-04-17T12:57:47.569Z
        created: 2022-03-18T12:57:47.576Z
  code:
    - some/path
    - some/file
  iac-drift:
     - '*'
     - '!aws_iam_*'
     - 'aws_s3_*'
     - 'aws_s3_bucket.*'
     - 'aws_s3_bucket.name*'
version: v1.22.2
patch: {}
```

the output:
<img width="1055" alt="Screenshot 2022-03-25 at 14 36 08" src="https://user-images.githubusercontent.com/6909300/160121914-6afa48ad-ab2b-4774-9bf7-bc5f3204c33a.png">


I also updated the tests to include the `iac-drift` entries.
Thanks a lot, @aron ! 🙏
